docs/user-guide/reading_writing_data.rst

```diff
@@ -140,17 +149,19 @@ The :py:class:`~hermes_core.timedata.TimeData` can the be updated, measurements
 Adding data to a ``TimeData`` Container
 =======================================
 
-A new column of data can be added to an existing instance.
-Remember that these new measurements must have the same time stamps as the existing ones and therefore the same number of measurements.
+A new column of data, support data, or NRV data can be added to an existing instance. Remember 
+that new data measurements must have the same time stamps as the existing ones and therefore 
+the same number of entries. Support data and non-record-varying data can be added as needed.
 You can add the new column in one of two ways.
 The more explicit approach is to use :py:func:`~hermes_core.timedata.TimeData.add_measurement` function::
 
     >>> data = u.Quantity(np.arange(len(timedata['Bx'])), 'Gauss', dtype=np.uint16)
     >>> timedata.add_measurement(measure_name="By", data=data, meta={"CATDESC": "Test Metadata"})
 
-Or you can just add the column directly.
+Or you can add the data, support, or NRV column directly.
 
     >>> timedata["By"] = u.Quantity(np.arange(len(timedata['Bx'])), 'Gauss', dtype=np.uint16)
+    >>> timedata["calibration_const"] = Column(data=[1e-3])
```

Comment:
I feel like treating NRV data and time series data with the same API could be very confusing. How about we add it as an attribute?

Reply:
I think I disagree, I would prefer using the same API for time-varying and non-time-varying data and then let the data container handle it under the hood. This is the same paradigm that `spacepy.pycdf` uses as well.